updates Gigabit T&C

hannahkane opened the pull request:
Fixes #2475

**Changes proposed in this pull request:**
- updates Gigabit Terms page as per [this doc](https://docs.google.com/document/d/1v4ORY4d3hvp1P8XH2x0b2KkxKbFcqihEIaxV2Fd-9m4/edit#)

hannahkane commented:
@mmmavis - sorry to throw another one at you; Gigabit team would like this updated as soon as we can, so they can start promoting

mmmavis commented on pages/gigabit/Terms.jsx:
```diff
@@ -22,120 +22,221 @@ module.exports = React.createClass({
+              </ol>
+              <li>Internet Health</li>
+              <ol>
+                <li>With which Internet health issue(s) does your project align? How does it help to build a healthier future internet?</li>
```
`Internet health issue` should be hyperlinked

mmmavis replied:
`Internet health issue` should be hyperlinked

mmmavis commented on pages/gigabit/Terms.jsx:
```diff
@@ -22,120 +22,221 @@ module.exports = React.createClass({
+              The following organizational information must be submitted along with your project description via mozilla.fluxx.io:
+            </p>
+            <ol>
+              <li>Required of all Non-Profit Applicants</li>
```
I think the `Required of ...` lines should be `<p`> and not bullet points.

```jsx
<p><u>Required of all Non-Profit Applicants</u></p>
<ol>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ol>

<p><u>Required of all For-Profit Applicants</u></p>
<ol>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ol>
```

It reads better this way. What do you think? @hannahkane

hannahkane commented:
Thanks, @mmmavis. I added the Internet Health issues link, changes the line items to paragraphs (agreed, that's better), and removed a line item from the Fund Reporting section. Not sure what I missed in the Legal Rights in Submission section, though. Can you clarify? Thank you!

mmmavis commented:
@hannahkane 

> Not sure what I missed in the Legal Rights in Submission section, though. Can you clarify?

Ah I didn't really read through the copy carefully. I checked things mostly by simply "copying the new text and searching it on the localhost page". Anyways, I just did a side-by-side comparison and the difference turned out to be something really tiny

`By entering a Submission, you agree to these Terms and Conditions and you represent and warrant that` <--- on the doc there's a comma before "and you represent and warrant that". I can't tell if that comma is absolutely needed since English is not my thing 😂 Will let you decide on that.

hannahkane commented:
good catch! Thanks, @mmmavis. 

If this looks ok to you, could you please merge and push to prod whenever you have time today? Thank you very much!
